iot_verifier/src/reward_share.rs

```diff
@@ -62,6 +64,16 @@ fn get_scheduled_dc_tokens(duration: Duration) -> Decimal {
     )
 }
 
+fn get_scheduled_ops_fund_tokens(duration: Duration) -> u64 {
+    get_tokens_by_duration(
+        *REWARDS_PER_DAY * *OPERATIONS_REWARDS_PER_DAY_PERCENT,
+        duration,
+    )
+    .round_dp_with_strategy(0, RoundingStrategy::ToZero)
```

Comment:
We use this rounding strategy to round any fractional bones right before we write the amounts to the output, because operation funds have no other calculations, we use this strategy here.